packages/training/python/scripts/test_judge.py

```
@@ -109,7 +109,21 @@ def evaluate_trajectory(traj: BabylonTrajectory):
 
 def main():
-    source_dir = "../engine/training-data-output/trajectories"
+    # Look for trajectory data in the training package output directory
+    source_dir = Path(__file__).parent.parent.parent / "training-data-output" / "trajectories"
+    if not source_dir.exists():
+        # Fallback to engine output if training output doesn't exist
+        source_dir = Path(__file__).parent.parent.parent.parent / "engine" / "training-data-output" / "trajectories"
+    
+    # Validate that at least one path exists
+    if not source_dir.exists():
+        logger.error(f"No trajectory data found. Checked paths:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)